Dockerfile

```diff
@@ -15,15 +15,7 @@ RUN apt-get update && \
         git \
         gnupg2 \
         default-jre \
-        python3-pip && \
-    apt-key adv --keyserver hkp://keyserver.ubuntu.com --recv 379CE192D401AB61 && \
-    echo "deb https://dl.bintray.com/kaitai-io/debian jessie main" \
-        | tee /etc/apt/sources.list.d/kaitai.list && \
-    apt-get update && \
-    apt-get install -y \
-        kaitai-struct-compiler && \
-    pip3 install kaitaistruct
-
+        python3-pip 
```

Comment:
I think you can drop `gnupg2`, `default-jre` and `python3-pip` as well now: they were only needed for the `apt-key` below and Kaitai.